src/explorer.api/Queries/DistinctColumnValues.cs

```diff
@@ -34,7 +35,7 @@ public DistinctColumnValues(string tableName, string columnName)
 
         public class IntegerResult : IJsonArrayConvertible
         {
-            public long? ColumnValue { get; set; }
+            public AircloakColumn<long> ColumnValue { get; set; }
```

Comment:
The compiler issues a warning here regarding the uninitialized non-nullable field (this is the same for all result types).

Reply:
☝️ Fixed